pkg/overlay/config.go

```diff
@@ -39,7 +39,7 @@ type NodeSelectionConfig struct {
 	AuditSuccessRatio float64       `help:"a node's ratio of successful audits" releaseDefault:"0.4" devDefault:"0"` // TODO: update after beta
 	AuditCount        int64         `help:"the number of times a node has been audited to not be considered a New Node" releaseDefault:"500" devDefault:"0"`
 	NewNodePercentage float64       `help:"the percentage of new nodes allowed per request" default:"0.05"` // TODO: fix, this is not percentage, it's ratio
-	MinimumVersion    string        `help:"the minimum node software version for node selection queries" default:""`
+	MinimumVersion    string        `help:"the minimum node software version for node selection queries" releaseDefault:"v0.10.1" devDefault:""`
```

Comment:
This version is gonna be weird to maintain.

Reply:
Whoever created this configuration probably didn't intend for it to be set in code.  Then again, if there is a cogent value, we should probably set it, right?

Reply:
yeah i think this is okay to leave unset